build-tools/scripts/JavaCallableWrappers.targets

```
@@ -16,7 +16,7 @@
       <_LibDirs>-L "$(OutputPathAbs.TrimEnd('\'))"</_LibDirs>
       <_LibDirs Condition=" '$(TargetFramework)' != 'net6.0' ">$(_LibDirs) -L "$(OutputPathAbs)..\v1.0" -L "$(OutputPathAbs)..\v1.0\Facades"</_LibDirs>
       <_LibDirs Condition=" '$(TargetFramework)' == 'net6.0' ">$(_LibDirs) -L "$(_CorlibDir.TrimEnd('\'))"</_LibDirs>
-      <_Out>-o "$(MSBuildProjectDirectory)\$(IntermediateOutputPath)jcw\src"</_Out>
+      <_Out>-o "$(MSBuildProjectDirectory)\$(IntermediateOutputPath)jcw/src"</_Out>
```

Maybe it's worth fixing all the slashes in this one:

```suggestion
      <_Out>-o "$(MSBuildProjectDirectory)/$(IntermediateOutputPath)jcw/src"</_Out>
```

It's weird this property doesn't have a trailing slash...

https://docs.microsoft.com/visualstudio/msbuild/msbuild-reserved-and-well-known-properties